github/actions_hosted_runners.go

```diff
@@ -0,0 +1,333 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
```

Comment:
For new files, we use the current year, then we never change the year of an existing file.

```suggestion
// Copyright 2025 The go-github AUTHORS. All rights reserved.
```